resources/core/charts/docs/charts/content-ui/templates/docs-components-compass-cdt.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui
+    cms.kyma-project.io/group-name: components
+    cms.kyma-project.io/order: "17"
```

Comment:
`cms.kyma-project.io/order: "16"`. Headless CMS is last and has 15 order.

Reply:
@magicmatatjahu Sure thing, thanks!